.pep8speaks.yml

```diff
@@ -0,0 +1,22 @@
+# File : .pep8speaks.yml
+
+message:  # Customize the comment made by the bot
+    opened:  # Messages when a new PR is submitted
```

Comment:
Strongly against spamming all PRs like this. Better just leave a comment when something goes wrong.

Reply:
Yeah, this is sort of the default, I'll replace it with an empty string.